wled00/wled_serial.cpp

```diff
@@ -69,9 +69,11 @@ void sendBytes(){
 
 void handleSerial()
 {
-  if (serialCanRX == false && Serial) {
-    return;
-  }; // arduino docs: `if (Serial)` indicates whether or not the USB CDC serial connection is open. For all non-USB CDC ports, this will always return true
+  #if ARDUINO_USB_CDC_ON_BOOT
+    if (!Serial) return;// arduino docs: `if (Serial)` indicates whether or not the USB CDC serial connection is open. For all non-USB CDC ports, this will always return true
+  #else
+    if (!(serialCanRX)) return; // check via pin manager if UART pin is NOT allocated by other function.
```

Comment:
to get the previous behaviour: either use the original line from WLED source code, or change it slightly to
````C++
     if (!serialCanRX || !Serial) return;
````